numba/core/config.py

```diff
@@ -223,6 +223,9 @@ def optional_str(x):
         # Enable debug prints in nrtdynmod and use of "safe" API functions
         DEBUG_NRT = _readenv("NUMBA_DEBUG_NRT", int, 0)
 
+        # Stack Traceback limit when DEBUG_NRT is enabled
+        DEBUG_NRT_STACK_LIMIT = _readenv("NUMBA_DEBUG_NRT_STACK_LIMIT", int, 3)
```

Comment:
Should the default be set to 0, to prevent existing users from getting unnecessary stack trace prints ?

Reply:
I think yes else it's altering a default behaviour. Most of the time NRT debugging is looking for a mismatched pair and it's often known where the inc/decs are coming from, this is adding a new capacity to help in the case where it's not obvious/really involved. Further, the use of `0` as default would mean it can just not be generated _at all_ by default so as to keep the IR smaller?